/*
 * app.cpp
 *
 *  Created on: Oct 2, 2018
 *      Author: bruno
 */

#include <iostream>
#include "exception.h"

using namespace std;

int main() {

	int a = 0;
	int b = 10;


try {
	if(a == 0) throw a;
}
catch(int error) {
	cout << "The variable cannot be equal to " << a << endl;
}
catch(...) {
	cout << "This is like an else statement. It will get every other types other than int"
}

	int div = b / a;
return 0;
}

